src/components/Passenger/Passenger.tsx

```diff
@@ -0,0 +1,44 @@
+import styles from "./Passenger.module.css";
+import { useState } from "react";
+import type { PassengerProps } from "./Passenger.definitions";
+
+/**
+ * Passenger component - a simple component that holds passenger details
+ * @property {string} firstName
+ * @property {string} lastName
+ * @property {string} email
+ * @property {string} relationship
+ * @property {string} userType
+ * @property {string} dateOfBirth
+ */
+const Passenger = (Passengers: PassengerProps) => {
+  const [number, setNumber] = useState<number>(0);
+
+  return (
+    <div className={styles.passengerCard}>
+      <div className={styles.title}>Passenger {}</div>
```

Comment:
Maybe we can make another component for these labels? How about something like:

```tsx
<CustomLabel title={ ... } color={ ... } />
```

This could be useful all over the app!

Reply:
How would you do this? Would the custom Label be in place of div?

Reply:
Yes! The `<CustomLabel ... />` component would be in place of the `<div ... />` tags. You can make this component just like we made this one! Make a new folder called `/CustomLabel` in the `/components` folder and give the files it needs (the `.tsx`, `.module.css`, and `.definitions.ts` files). Then you can create the component! Since this is just a label it should be **super** simple, _hint: you might only need just one div for this entire component_.

Let's give the component **3 props**: `title`, `backgroundColor`, and `textColor`. All three of these will be strings. The title will be the text in the `<div>` tags and the two colors can be used for the styles. Instead of giving the div it's color in the `.css` file, we can inject some styles in the typescript code like this:

```tsx
<div styles = {{
    backgroundColor: BACKGROUND_COLOR_VARIABLE,   // color the background
    color: TEXT_COLOR_VARIABLE,  // color the text
    border: `3px solid ${TEXT_COLOR_VARIABLE}` // create and color the border
}}> 
  ....something goes here...
</div>
```

_Note: Those all-capital variable names should be replace with whatever the variable name is for those properties_

**Let's take it one step further!**

When we define the props for our new component, the `title`, `backgroundColor`, and `textColor` are all going to be of type string! However we know that the colors are going to be hex codes (example: `"#2FE4A9"`). Hex codes are just a way of representing color in text. We can limit our types to **only accept strings that are prefixed with a `#`**. We can do that like this:

```ts
export interface SomethingSomethingProps {
// ...
colorThing: `#${string}`
// ...
}
```
_Note: Those are back-ticks, not single quotes!_

Now the property `colorThing` can only be a type string that is prefixed by a `#`! This is just one of the many many cool things we can do with typescript.